stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -258,7 +258,7 @@ julia> subtypes(Integer)
 subtypes(x::Type) = _subtypes_in(Base.loaded_modules_array(), x)
 
 # dumptype is for displaying abstract type hierarchies,
-# based on Jameson Nash's examples/typetree.jl
+# based on Jameson Nash's typetree.jl in JuliaArchive/Examples
```

Comment:
`... in the GitHub repository JuliaArchive/Examples`.  Otherwise I might think this is a folder within this repository.